pkg/generate/generator.go

```diff
@@ -223,6 +223,9 @@ func (g *Generator) GetCRDs() ([]*ackmodel.CRD, error) {
 			}
 		}
 
+		// TODO(nithomso): Is this the correct casing?
+		crd.NameField = crd.GetNameField()
```

Comment:
Yeah this is unclean. This needs to be called after all the fields have been added, though. Thoughts on better method?